raiden/tests/api/test_api.py

```diff
@@ -549,8 +576,12 @@ def test_query_blockchain_events(
     # of `get_channel_events()` but just makes sure the proper data make it there
     api_test_context.specify_channel_for_events('0xedbaf3c5100302dcdda53269322f3730b1f0416d')
     request = grequests.get(
-        'http://localhost:{port}/api/1/events/channels/0xedbaf3c5100302dcdda53269322f3730b1f0416d?from_block=10&to_block=90'.format(
-            port=rest_api_port_number,
+        api_url_for(
+            api_backend,
+            'channeleventsresource',
```

Comment:
It needs to be in lowercase. `url_for` works by lowercasing the resource class name which in this case is `ChannelEventsResource`